src/Api/Issue/GithubIssueApi.php

```diff
@@ -44,6 +44,14 @@ public function open(Repository $repository, string $title, string $body, array
         }
     }
 
+    public function lastCommentWasMadeByBot(Repository $repository, $number): bool
+    {
+        $allComments = $this->issueCommentApi->all($repository->getVendor(), $repository->getName(), $number);
```

Comment:
Since this is not using pagination. We will only fetch 100 comments. That means that we will never auto close issues with more that 100 comments. 

Im fine with that atm.